recipe/meta.yaml

```diff
@@ -28,12 +28,11 @@ test:
     - nose-cov
     - chai
     - simplejson
-  commands:
-    - nosetests "{{ environ['SRC_DIR'] }}"
```

Comment:
IIRC from our meeting the other day, we need to use [`source_files`]( http://conda.pydata.org/docs/building/meta-yaml.html#source-files ).